riff-raff/app/deployment/filter.scala

```diff
@@ -36,7 +38,10 @@ case class DeployFilter(
 
   lazy val sqlParams: List[SQLSyntax] = List(
     projectName.map { pn =>
-      val likeString = s"%$pn%"
+      val likeString = isExactMatchProjectName match {
+        case Some(true) => pn
```

Comment:
no quotes needed?

Reply:
ahhh so its using `sqls` template string interpreter... https://github.com/guardian/riff-raff/blob/fff87f9547be7fbaadf0cb3ef45e7b757a09b003/riff-raff/app/deployment/filter.scala#L45 

Which according to their documentation...
> Don’t worry again, this code is safely protected. sqls"" always treats external input values as binding parameters.
http://scalikejdbc.org/documentation/sql-interpolation.html